### PR TITLE
py-pandoc

### PR DESCRIPTION
**Pull request by kiwi0fruit**
`pandoc` should be pure conda package. But then pip in Anaconda is not aware of it.  
So `py-pandoc` is a wrapper around it that pip is aware of.

### CODE REVIEW

**Comment by kiwi0fruit** [comment]
@conda-forge/help-python please review.

**Comment by kiwi0fruit** [comment]
@conda-forge/help-python please review

File: recipes/py-pandoc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "py-pandoc" %}
+{% set version = "2.5" %}
+{% set pypi_build = ".0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
```
Comment (beckermr) [inline recipes/py-pandoc/meta.yaml]: Ack. I should have asked this in the other one maybe. Why is conda not getting the full version string here?

Comment (kiwi0fruit) [reply recipes/py-pandoc/meta.yaml]: @beckermr I guess you are right. And it's better to have a full version. I ve fixed. Please PR.

**Comment by kiwi0fruit** [comment]
@beckermr Version is fixed and tests passed. Please PR.

**Comment by kiwi0fruit** [comment]
@beckermr Version is fixed and tests passed. Please PR.

**Comment by kiwi0fruit** [comment]
@beckermr All tests passed. And thanks!

**Comment by beckermr** [comment]
Great. Merging now. Let’s make sure to fix the other package I flagged.

**Comment by kiwi0fruit** [comment]
@beckermr I've already fixed py-mathjax

**Comment by beckermr** [comment]
Great! thank you @kiwi0fruit!

**Pull request merged**